yarn.lock

```diff
@@ -1989,13 +1989,6 @@
   dependencies:
     regenerator-runtime "^0.13.4"
 
-"@babel/runtime@^7.16.7":
```

Comment:
These removals are a result of https://github.com/jbx-protocol/juice-contracts-v2/pull/264